Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/OAuthRepository/OAuthRepository.cs

```diff
@@ -56,7 +56,7 @@ private OAuth OAuthClientChecking(string email,string clientId)
         {
             //checking whether with this app email is register or not if  not new OAuth will be created.
             var oAuth = GetDetails(email, clientId);
-            var app = _appRepository.GetAppDetails(clientId);
+            var app = _appRepository.GetAppDetails(clientId).Result;
```

Comment:
because given method "OAuthClientChecking" is not an asynchronous method so i can not make it await. if you want then i will update this method